playbooks/roles/rabbitmq/defaults/main.yml

```diff
@@ -17,9 +17,9 @@ RABBIT_USERS:
   - name: 'admin'
     password: "{{ RABBIT_ADMIN_PASSWORD }}"
   - name: 'edx'
-    password: 'edx'
+    password: "{{ XQUEUE_RABBITMQ_PASS }}"
   - name: 'celery'
-    password: 'celery'
+    password: "{{ EDXAPP_CELERY_PASSWORD }}"
```

Comment:
Where is the source of this variables? If they are undefined, we will get error. Maybe `"{{ var|default('some pass') }}"` ?